extension/js/content_scripts/webmail/gmail_element_replacer.ts

```diff
@@ -221,7 +221,7 @@ export class GmailElementReplacer implements WebmailElementReplacer {
     for (const a of attMetas) {
       const treatAs = a.treatAs();
       // todo - [same name + not processed].first() ... What if attachment metas are out of order compared to how gmail shows it? And have the same name?
-      const attSel = this.filterAtts(attsContainerInner.children().not('.attachment_processed'), [a.name]).first();
+      const attSel = this.filterAtts(attsContainerInner.children().not('.attachment_processed'), new RegExp(a.name)).first();
```

Comment:
I think you may need to regex-escape. I suspect this will fail with a filename `something[hello].txt`. Maybe it would even throw exception on bad regex like `some[thing.txt`, not sure. You can test it to see for yourself, and then probably use `Str.regexEscape` on the string before passing it to `new RegExp()`. After escaping, you could also add `^` on the beginning and `$` at the end just to be sure.

Reply:
oh, yes, I forgot that these situations could have happened. 
Thanks!